docs/pages/project/contributing/contributing-docs.md

```diff
@@ -147,6 +147,25 @@ Meshery documentation is made of these components:
 
   _Note: From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`. There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when you need to generate the site artifacts for production._
 
+#### Note
+While performing the above step, if you're facing errors with a message like below...
+
+`Your ruby version is x.x.x but your Gemfile specified 2.7.x`
+
+This is because Jekyll always considers the exact version of Ruby unlike JavaScript.
+
+So, you need to follow either of the two steps to resolve this problem;
+  - Install the required Ruby version by using `rvm` or by any means given above
+  - If you're unable to install the required Ruby version, then manually configure the `Gemfile` as below;
+  ```
+  source "https://rubygems.org" 
+  ruby '2.7.1' //to any version you have installed
+  ```
+  Automatically the `Gemfile.lock` will update once the `make site` is given (for Windows, run `bundle exec jekyll serve` alongside `make site`)
```

Comment:
For Windows, I guess we need to run anyone `make site` or `bundle exec jekyll serve` we do not need to run both, also without wsl2 `make site` doesn't run so well, if wsl2 is not integrated, then `bundle exec jekyll serve` does the job!

Reply:
Hmm alright!

Reply:
Changes made